File: Python/2021/1stOct_abhishekpandey1880.py
# Made by Abhishek
import math
def isPrime(x):
    if x <= 1:
        return False
    if x == 2:
        return True
    if x > 2 and x % 2 == 0:
        return False
 
    end = math.floor(math.sqrt(n))
    for i in range(3, end+1, 2):
        if x % i == 0:
            return False
    return True


n = int(input("Enter number of elements: "))
sum = 0
n1 = 1
n2 = 1
count = 0
if n == 1:
  print(n1)
else:
  while(count<n):
    if (not isPrime(n1)) and (n1 % 5 != 0):
      print(n1, end = ' ')
    else:
      print(0, end = ' ')
    sum = n1 + n2
    n1 = n2
    n2 = sum
    count += 1

File: JAVA/2021/1stOct_abhishekpandey1880.java
import java.util.Scanner;
class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int n, n1 = 0, n2 = 1, n3 = 0;

        System.out.print("Enter the number of elements: ");
        n = input.nextInt();;

        if (n == 1)
            System.out.println(n2);
        for (int i = 1; i <= (n + 1); ++i) {

            n3 = n1 + n2;
            n1 = n2;
            n2 = n3;
            if (isPrime(n3) || (n3 % 5 == 0))
                System.out.print("0" + " ");
            else
                System.out.print(n3 + " ");
        }
    }

    public static boolean isPrime(int n) {
        boolean ans = true;
        if (n == 0 || n == 1) {
            ans = false;
        } else {
            for (int i = 2; i <= n / 2; ++i) {
                if (n % i == 0) {
                    ans = false;
                    break;
                }
            }
        }
        return ans;
    }
}
